### PR TITLE
Tuning model3 again

### PR DESCRIPTION
**Pull request by mithunpaul08**

### CODE REVIEW

**Comment by mithunpaul08** [comment]
have added a keras.tuner for model 3. It is now capable of tuning across various parameters like activation, loss etc, in a grid search fashion. Note that this code wasnt able to produce any improvements in the food-it dataset, but then we thought, why waste time trying to tune into a toy dataset, why not start with a good english dataset itself. So merging the code in so that we can move on with a standard dataset like GLUE or something
